Hermes/appscale/hermes/stats/producers/proxy_stats.py

```diff
@@ -286,14 +286,15 @@ def get_frontend_ip_port(configs_dir, proxy_name):
     for line in proxy_conf:
       line = line.strip()
       if line.startswith('bind'):
-        return line.split(' ')[1].split(':')
+        ip, port = line.split(' ')[1].split(':')
+        return ip, int(port)
```

Comment:
Was it previously counting 0 connections since it was comparing string vs int?

Reply:
yes=(